src/mono/mono/mini/method-to-ir.c

```diff
@@ -9546,6 +9558,63 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 				}
 			}
 
+			// ASSUME
+			//   interface ISomeIface { void Method(); }
+			//   struct SomeStruct : ISomeIface {...}
+			// OPTIMIZE
+			//   box SomeStruct
+			//   callvirt instance ISomeIface::Method()
+			// TO
+			//   call SomeStruct::Method()
+			guchar* callvirt_ip;
+			guint32 callvirt_proc_token;
+			if((callvirt_ip = il_read_callvirt(next_ip, end, &callvirt_proc_token)) && 
+				ip_in_bb(cfg, cfg->cbb, callvirt_ip) ) 
+			{
```

Comment:
formatting: there are several small issues regarding formatting, for example: 
- mono C source follows k&r indentation style
- space between function name and braces
- space after if

Here is a useful guideline: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/mono-code-guide.md